README.md

```diff
@@ -79,7 +79,36 @@ from .env file, Otherwise app will not work locally.
 
 ## Installing the bot on a repository
 
-You'll need to identify a target repository and install the bot by clicking the **Install** button on the settings page of your app, e.g `https://github.com/settings/apps/my-oppiabot-testing-app/installations`. In the `.env` file put your github account name in `WHITELISTED_ACCOUNTS` and also add your repository (name in small caps) in the `constants.js` file locally.
+1. Identify a target repository and install the bot by clicking the Install button on the settings page of your app, e.g., **`https://github.com/settings/apps/my-oppiabot-testing-app/installations`** 
+2. Update the **`.env` file**:
+   - Add your GitHub account name to `WHITELISTED_ACCOUNTS`.  
+3. Add the repository name in the **`constants.js` file**:  
```

Comment:
There is a gap of only one line after every heading. Like:
## Installing the bot on a repository

1. Identify a target repository and install the bot by clicking the Install button on the settings page of your app, e.g., **`https://github.com/settings/apps/my-oppiabot-testing-app/installations`** 
2. Update the **`.env` file**...

## Running the bot locally

You are now ready to run the bot on your local machine. Run `npm run dev` to start the server.

So, I followed the same

Reply:
No, I mean a new line above "2." and above "3.". I think that this won't change how the formatted text looks, but it will make the md file a bit easier to read.

Reply:
Okay, I have made the changes suggested by you.